demo.md

```diff
@@ -0,0 +1,23 @@
+## Demo script:
+
+- ERC-4337 compatible Smart Contract wallet
+- Unique feature: gasless off-chain txs with instant finality enabled by HTLC technology, as used in Bitcoin Lightning. Funds can be moved in and out of the off-chain payment system without any on-chain txs!
```

Comment:
We may want to be careful with the term `gasless`. While it's true the off-chain payments don't require gas, it could get confused with "gasless" ERC 4337 transactions, which is a whole different kettle of fish.

Reply:
But our txs are actually `gasless` whereas the ERC-4337 txs are more like "sponsored txs" where somebody else pays the gas 🙄. You are correct though, might be confusing to use that term. Originally I had "free off-chain txs". Do you like that better or can you think of any other term that might be better?

Reply:
Something like that makes sense to me! Maybe "free off-chain payments"? The term`Off-chain tx` seems like it might be slightly confusing, since `on-chain tx` let you do much more then pay someone.

Reply:
Agreed on the use of "off-chain payment" instead of "off-chain tx".

"free payment" seems contradictory but I'm good with it if you are 🤷‍♂️